src/jupyter_contrib_nbextensions/nbextensions/load_tex_macros/readme.md

```diff
@@ -0,0 +1,4 @@
+Usage
+=====
+
+Simply put your latex macros in a file named latexdefs.tex, in the same directory as your notebook.
```

Comment:
Perhaps "This is mainly derived from [jupyter_latex_envs](https://pypi.python.org/pypi/jupyter_latex_envs), following the discussion [here](https://github.com/jupyter/notebook/issues/3358)".

Reply:
Sorry! didn't read the mention at the beginning. Thanks.